src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String STRING_SEPARATOR = ",";
+    private static final String[] REPORT_COLUMNS = new String[] {"supply", "buy", "result"};
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(fromFileName, toFileName);
+    }
+
+    private void writeFile(String fromFileName, String toFileName) {
+        HashMap<String, Integer> results = calculateStatisticResults(fromFileName);
+
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            for (String column: REPORT_COLUMNS) {
+                bufferedWriter.write(column + "," + results.get(column) + System.lineSeparator());
```

Comment:
Use StringBuilder to avoid creating new String during concatenation
```suggestion
                bufferedWriter.write(column + COMMA + results.get(column) + System.lineSeparator());
```